app/src/main/java/net/gsantner/opoc/model/GsSharedPreferencesPropertyBackend.java

```diff
@@ -71,17 +71,14 @@ public class GsSharedPreferencesPropertyBackend implements GsPropertyBackend<Str
     protected String _prefAppName;
     protected Context _context;
 
-    public GsSharedPreferencesPropertyBackend() {
+    public GsSharedPreferencesPropertyBackend(final Context context) {
```

Comment:
I understand that you prefer AppSettings to instantiate with Context directly. But this here can be left with non-Context requiring constructor, doesn't it?